l10n_ch_payment_slip/payment_slip.py

```diff
@@ -483,9 +483,6 @@ def _draw_address(self, canvas, print_settings, initial_position, font,
         text.moveCursor(0.0, font.size)
 
         address_lines = com_partner.contact_address.split("\n")
-        if com_partner.country_id:
```

Comment:
@eLBati version 8.0 is quite old now, so I would prefer not to break something working.

So it would be better with backward compatibility.

This fix seems to have been applied only in 8.0 and didn't made it in 9.0
https://github.com/OCA/l10n-switzerland/commit/27d8fa88152927d21bf6cd14be709b15034e21f0

as it landed after the portage to 9.0

@johannbelet Any thought about this ?